docs/docs/how-to/images-and-media/using-gatsby-plugin-image.md

```diff
@@ -192,6 +192,30 @@ If you need to have dynamic images (such as if they are coming from a CMS), you
    `
    ```
 
+## Using images from a CMS
+
+Many CMS source plugins have native support for `gatsby-plugin-image`. The images are served directly from a content delivery network. This means that builds are faster, because there is no need download images and process them locally at build time. The query syntax varies according to the plugin, as do the supported transformation features and image formats. Make sure you update to the latest version of the source plugin to ensure there is support.
+
+### Source plugins
+
+These source plugins support using `gatsby-plugin-image` with images served from their CDN.
+
+- [Contentful](/plugins/gatsby-source-contentful/#using-the-new-gatsby-image-plugin)
+- [DatoCMS](/plugins/gatsby-source-datocms/#integration-with-gatsby-image)
+- [Shopify](https://github.com/gatsbyjs/gatsby-source-shopify-experimental#images)
+- [AgilityCMS](https://github.com/agility/gatbsy-image-agilitycms)
+- [Sanity](https://www.gatsbyjs.com/plugins/gatsby-source-sanity/#using-images)
```

Comment:
Small remark - sort by alphabetical order so there can be no dispute on who goes first and who goes second.

Reply:
Good point. They were in the order support was added. Here's a PR updating it: #30327